test/honeydew/queue/ecto_poll_queue_integration_test.exs

```diff
@@ -1,22 +1,49 @@
 defmodule Honeydew.EctoPollQueueIntegrationTest do
+  alias Mix.Shell.IO, as: Output
   use ExUnit.Case, async: false
   @tag timeout: 2 * 60 * 1_000
   @examples_root "./examples/ecto_poll_queue"
-  @databases ~w(postgres cockroach)
-
-  Enum.each(@databases, fn database ->
-    test "ecto poll queue external project test: #{database}" do
-      database = unquote(database)
-      IO.puts("\n")
-      IO.puts("------- Ecto Poll Queue: #{database} -------")
-      mix "deps.get", database
-      mix "test", database
-      IO.puts("-------------------------------")
-    end
-  end)
 
-  defp mix(task, database) do
-    {_, exit_code} = System.cmd("mix", [task], cd: @examples_root, into: IO.stream(:stdio, 1), env: [{"MIX_ENV", database}])
+  # Postgres
+  test "ecto poll queue external project test: Postgres" do
+    announce_test("Postgres (unprefixed)")
+    mix("deps.get", "postgres", prefixed: false)
+    mix("test", "postgres", prefixed: false)
+  end
+
+  test "ecto poll queue external project test: Postgres (prefixed)" do
+    announce_test("Postgres (prefixed)")
+    mix("deps.get", "postgres", prefixed: true)
+    mix("test", "postgres", prefixed: true)
+  end
+
+  # Cockroach
+  test "ecto poll queue external project test: Cockroach" do
+    announce_test("CockroachDB (unprefixed)")
+    mix("deps.get", "cockroach", prefixed: false)
+    mix("test", "cockroach", prefixed: false)
+  end
+
+  defp announce_test(message) do
+    Output.info(
+      "\n#{IO.ANSI.underline()}[ECTO POLL QUEUE INTEGRATION] #{message}#{IO.ANSI.reset()}"
+    )
+  end
+
+  defp mix(task, database, prefixed: prefixed_tables) do
+    environment = [{"MIX_ENV", database}] |> add_prefixed_tables_env(prefixed_tables)
+
+    {_, exit_code} =
+      System.cmd("mix", [task], cd: @examples_root, into: IO.stream(:stdio, 1), env: environment)
+
     assert exit_code == 0
   end
+
+  defp add_prefixed_tables_env(env, prefixed_tables) do
```

Comment:
instead of an explicit conditional here, what do you think of doing the matching in the function head instead? like

```
defp add_prefixed_tables_env(env, true) do
  env ++ [{"prefixed_tables", "true"}]
end

defp add_prefixed_tables_env(env, false), do: env
```

Reply:
SGTM! If you want you can just push it if you're on it, otherwise I can make the change tomorrow as well 👍